src/btf.cpp

```diff
@@ -387,6 +387,8 @@ SizedType BTF::get_stype(const BTFId &btf_id, bool resolve_structs)
     } else {
       stype = CreateArray(array->nelems, elem_type);
     }
+  } else if (btf_is_type_tag(t)) {
```

Comment:
I'm conflicted whether or not we should allowlist certain known type tags. For example, rcu type tag can safely be ignored while percpu type tag cannot. The latter requires special helpers to access percpu memory.

On the other hand, only clang-built-linux will have vmlinux type tags. And clang-built-linux is not widely adopted in open source land.

Perhaps a comment calling this out will suffice for now?